docs/packaging/packaging-changes.md

```diff
@@ -0,0 +1,51 @@
+# Packaging Changes
+
+This document is meant as a changelog of sorts for packaging practices. Refer to specific help pages for more details.
+
+- This list is not exhaustive
+- This list is in _reverse chronological order_ (newest first)
+
+## Upcoming
+
+_These items are not mandatory yet, and are subject to change_
+
+#### `makefile` -> `go-task`
+
+- All packaging action are now handled by `go-task` rather than `make`.
+  - Example: `make` to build a package is replaced with `go-task build`.
+  - Most commands are otherwise the same: `make local`, for example, is replaced by `go-task local`.
+  - You can see all available commands at `common/Taskfile.yml`
+- Packagers should install `go-task` on their machines if they have not already
+- The `Makefile` included with every package is no longer required. Please delete it from a package when updating, and do not include it when making the initial commit of a new package
```

Comment:
I personally prefer a paragraph instead of bullet points (also applies to the rest). Also: `go-task` is the equivalent of `build`:

```suggestion
All packaging action are now handled by `go-task` rather than `make`. Building a package, for example, can be done using `go-task` instead of `make`. Commands are the same: `make local`, for example, is replaced by `go-task local`.
You can see all available commands at `common/Taskfile.yml`
Packagers should install `go-task` on their machines if they have not already
The `Makefile` included with every package is no longer required. Please delete it from a package when updating, and do not include it when making the initial commit of a new package.

Reply:
This is the only one I disagree with; I prefer the heavily bulleted style. But I'll rewrite as paragraphs if others prefer it too